common/src/escrow_accounts.rs

```diff
@@ -13,18 +13,48 @@ use tracing::{error, warn};
 
 use crate::prelude::{Query, SubgraphClient};
 
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
+pub struct EscrowAccounts {
+    pub balances: HashMap<Address, U256>,
+    pub signers_to_senders: HashMap<Address, Address>,
```

Comment:
No. It's very important that there is only a single sender behind a signer, so that we always know without ambiguity who to charge just from the receipt signature.
It is enforced by the contracts there:
- https://github.com/semiotic-ai/timeline-aggregation-protocol-contracts/blob/cbc9a5acd3401700bf482ccc24ad05ab339614cd/src/Escrow.sol#L45-L47
- https://github.com/semiotic-ai/timeline-aggregation-protocol-contracts/blob/cbc9a5acd3401700bf482ccc24ad05ab339614cd/src/Escrow.sol#L505

Reply:
That makes sense, thanks!